### ISSUE TITLE
Make Linux and Windows build

### ISSUE THREAD

**Issue by bolasim** [nathanbirrell/overcast-macos#3]
Since the app is developed on Electron, this should be rather straight forward. Any reason not to do this?

I use Linux daily but would love to use this app.

**Comment by nathanbirrell** [comment]
Yeah it should be, my initial focus has been macOS simply because that's what I use. Will look into this..

**Comment by nathanbirrell** [comment]
Hey @coptic-bola, I don't think I'll get time to do and test this properly, do you want to then create a PR if you can?
